modules/api/src/main/java/org/apache/ignite/schema/ColumnType.java

```diff
@@ -116,6 +119,37 @@ public static NumericColumnType number(int precision, int scale) {
         return new NumericColumnType(ColumnTypeSpec.DECIMAL, precision, scale);
     }
 
+    /**
+     * Timezone-free value representing a time of day in hours, minutes, seconds,
+     * and fractional seconds depending on type precision.
+     *
+     * @param precision Fractional seconds part length. Allowed values are 3/6/9 for millis/micros/nanos.
```

Comment:
Why we have such restrictions? For example, Postgresql could use any precision between 0 and 6, in case the user set more than 6 then use 6